README.md

```diff
@@ -33,8 +59,8 @@ The apostrophe-cli assumes the `apostrophe` namespace when executing tasks. This
 apostrophe <task name>
```

Comment:
"The apostrophe-cli assumes the `apostrophe` namespace when executing tasks" so you could run apostrophe taskname if the task you wanted to run was on the apostrophe-namespace... You would have to <namespace>:<task name> if you wanted to run something on another namespace. I added a point clarifying this